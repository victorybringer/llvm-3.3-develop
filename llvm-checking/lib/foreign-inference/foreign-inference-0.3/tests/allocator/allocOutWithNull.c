#include <stdlib.h>

void outAlloc(int n, int** arr) {
  if(n == 0)
    *arr = NULL;
  else
    *arr = malloc(n * sizeof(int));
}
