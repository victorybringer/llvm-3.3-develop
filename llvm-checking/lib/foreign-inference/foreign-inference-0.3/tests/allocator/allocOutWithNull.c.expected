(fromList [], fromList [ ("outAlloc", fromList [("arr", PAOutAlloc "free")]) ])
